arbitrum-docs/for-devs/concepts/public-chains.mdx

```diff
@@ -30,15 +29,25 @@ Arbitrum Sepolia serves as a testnet chain replicating the capabilities of Arbit
 
 Arbitrum Goerli is a testnet chain that mirrors the functionality of the Arbitrum One mainnet. It's connected to the Ethereum Goerli testnet, allowing developers to test their smart contracts in a risk-free environment before deploying them on the mainnet.
 
-⚠️ Important note: Ethereum Goerli was deprecated in Q1 2023 but will be supported until Q1 2024. We are currently phasing out the usage of Arbitrum Goerli. Support for Arbitrum Goerli will cease prior to the conclusion of Ethereum Goerli.
+:::caution 
```

Comment:
Bit of boyscouting here - a handful of docs were using emojis to communicate caution. Docusaurus natively supports [admonitions](https://docusaurus.io/docs/markdown-features/admonitions), so we can use those.